spec/rspec/core/configuration_spec.rb

```diff
@@ -312,12 +312,10 @@ def mod.configuration; @config ||= Struct.new(:custom_setting).new; end
       it_behaves_like "a configurable framework adapter", :mock_with
 
       it "allows rspec-mocks to be configured with a provided block" do
-        mod = Module.new
-
-        expect(RSpec::Mocks.configuration).to receive(:add_stub_and_should_receive_to).with(mod)
+        expect(RSpec::Mocks.configuration).to receive(:verify_partial_doubles=).with(true)
 
         config.mock_with :rspec do |c|
-          c.add_stub_and_should_receive_to mod
```

Comment:
Why was this changed?

Reply:
`add_stub_and_should_receive_to` is no more.

Since the example is titled "allows rspec-mocks to be configured with a provided block", I've used one of the remaining `RSpec::Mocks::Configuration` options.

Reply:
Oh, silly me, I was reading this as changing from an imaginary config to an existing thing but thats not it at all. Carry on 😂